plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/StandardDeployPreferencesPanel.java

```diff
@@ -304,13 +301,9 @@ private void createProjectVersionSection() {
 
     overrideDefaultVersionButton = new Button(versionComposite, SWT.CHECK);
     overrideDefaultVersionButton.setText(Messages.getString("use.custom.versioning"));
-    overrideDefaultVersionButton.setLayoutData(new GridData(SWT.LEFT, SWT.CENTER, true, false, 2, 1));
-
-    versionLabel = new Label(versionComposite, SWT.NONE);
-    versionLabel.setText(Messages.getString("project.version"));
     GridData layoutData = GridDataFactory.swtDefaults().create();
-    layoutData.horizontalIndent = INDENT_CHECKBOX_ENABLED_WIDGET;
-    versionLabel.setLayoutData(layoutData);
+    layoutData.horizontalAlignment = GridData.BEGINNING;
```

Comment:
Not an export of SWT, but the Javadoc of `swtDefaults()` says `BEGINNING` is the default. Still need this?